src/librustc_ast/attr/mod.rs

```diff
@@ -173,8 +173,9 @@ impl Attribute {
         }
     }
 
-    /// Returns `true` if the attribute's path matches the argument. If it matches, then the
-    /// attribute is marked as used.
+    /// Returns `true` if the attribute's path matches the argument.
+    /// If it matches, then the attribute is marked as used.
+    /// Should only be used by rustc, other tools can use `has_name` instead.
```

Comment:
Can you expand this comment slightly, to explain that other tools don't do error reporting relating to attribute use?

Reply:
Done.